github/repos_projects.go

```diff
@@ -59,9 +63,9 @@ func (s *RepositoriesService) ListProjects(owner, repo string, opt *ListOptions)
 
 // GetProject gets a GitHub Project for a repo.
 //
-// GitHub API docs: https://developer.github.com/v3/repos/projects/#get-a-project
-func (s *RepositoriesService) GetProject(owner, repo string, number int) (*Project, *Response, error) {
-	u := fmt.Sprintf("repos/%v/%v/projects/%v", owner, repo, number)
+// GitHub API docs: https://developer.github.com/v3/projects/#get-a-project
+func (s *RepositoriesService) GetProject(id int) (*Project, *Response, error) {
+	u := fmt.Sprintf("%s/%v", projectUrl, id)
```

Comment:
This method and ones below should be a part of a new `ProjectsService`, since they're under a different endpoint and no longer related to `RepositoriesService`.